src/app/modules/main/chat_section/chat_content/messages/module.nim

```diff
@@ -343,7 +343,7 @@ method messagesAdded*(self: Module, messages: seq[MessageDto]) =
     # https://github.com/status-im/status-desktop/issues/7632 will introduce deleteFroMe feature.
     # Now we just skip deleted messages
     if message.deleted or message.deletedForMe:
-      return
+      continue
```

Comment:
Can you create a new PR with just this change, so we can have that merged at least for now, until we figure out what to do with the status-go side. Thanks

Reply:
https://github.com/status-im/status-desktop/pull/9529